Advanced Animator Parser (#343)
anatawa12/AvatarOptimizer: CHANGELOG-PRERELEASE.md

## [Unreleased]
### Added

> - Support for Layered Animator Parser `#343`

Author: Advanced Animator Parserに書き換えて`Advanced Animator Parser`の説明数行入れる

Author: 057f9c506bd8c5b5da7bf903df80a4155991fad3

## [Unreleased]
### Added
- Support for Multi Frame BlendShapes `#333`
- Advanced Animator Parser `#343`
  - This is new AnimatorController parser to collect animated properties
  - This parser understands AnimatorLayers so with this parser,

>     AAO can freeze BlendShapes which always be animated to constant value on the final later.

Reviewer: ```suggestion
    AAO can freeze BlendShapes which are always finally animated to a constant value.
```
